src/components/tasks/card/Loader.tsx

```diff
@@ -0,0 +1,9 @@
+import classNames from '@/components/tasks/card/card.module.scss';
```

Comment:
We should add unit tests for these components as well.

Reply:
done